### PR TITLE
Awesome Book Classes

### PR DESCRIPTION
**Pull request by Chimoneg27**
### For this feature I did the following

- Removed linter errors
- Add styles for the book div
- Made use of classes for my object constructor
- Ensured that my code functions well, even after the  refactoring 
- Made use of array of objects in classes

**Important note**

**My coding partner did not show up today and yesterday. So I have been given permission to continue the rest of the week alone. See the screenshot below**

Screenshot 2023-08-09 204149

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -4,27 +4,54 @@
...
+  margin: 1rem auto;
   display: flex;
   flex-direction: column;
   width: 15rem;
   height: auto;
-  align-items: flex-start;
+  align-items: flex-end;
   gap: 0.5rem;
   padding: 2rem;
 }
```
Comment (Alusp) [inline styles.css:25]: - [ ] Good job so far, :clap: , However, please ensure that your styling matches   given requirement as much as possible, matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

Below is an illustration of yours and the given Design, Please consider the red painting in  your  design :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

Student Design


</td>
      <td>

awesome_books_core_elements

</td>
    </tr>
  </table>

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @Chimoneg27, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
 
 close

 ## Highlights

- [ ] Linters set-up ✔️
- [ ] Commit messages descrptive ✔️
- [ ] Professional Github Flow used ✔️
- [ ] Professional PR Summary Flow used ✔️
- [ ] Create  Class :heavy_check_mark: 
- [ ] Add  and Remove  is successful  :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nicupop729 — COMMENTED** [review]
Hello @Chimoneg27  👋,

_This is an additional review as a result of the second opinion request._

Great job on this project! :clap:
The code reviewer's request isn't necessary for this milestone, and since all project requirements have been met, the CRR is ready for approval.

Here are my optional suggestions:
- Create a function to clear input fields after form submission (currently, they remain populated with the previous entry).
- Minimize the use of deactivating linters' comments as much as possible.

Since that was the only reason your project wasn't approved, it will be approved as soon as possible.
Please request another review on your Dashboard, and the next Code Reviewer will approve it immediately :)

Keep rocking 🚀!

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Garvin,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 🎯 
🎲 Good job on implementing classes in your code!

## Optional suggestions 🧰 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

- [ ] [OPTIONAL]
"Create a function to clear input fields after form submission (currently, they remain populated with the previous entry)."
- [ ] [OPTIOANL]
"Minimize the use of deactivating linters' comments as much as possible."

Cheers..Happy coding! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**